### DISCUSSION TITLE
AUTOGEN AttributeError

### DISCUSSION THREAD

**Discussion by ray30087** [microsoft/autogen#1535] [Q&A]
I'm running this autogen exercise, but it shows an attritube error that I'm so confused...
really need some instructions, please~~
![image](https://github.com/microsoft/autogen/assets/32152701/f8a4f356-72be-4869-98e3-811b537583a3)

**Comment by sonichi** [comment]
Please uninstall "autogen" and install "pyautogen" if you installed the wrong package by chance.

**Reply by ray30087** [reply]
Thanks, i will try it though!

**Reply by ray30087** [reply]
I've install the correct package, but the same error still occur...
![Uploading image.png…]()

**Reply by ekzhu** [reply]
Do you have a folder in your working directory called "autogen". Please also make sure you have removed the python package `autogen`

**Reply by ray30087** [reply]
Do u mean this?
![image](https://github.com/microsoft/autogen/assets/32152701/a6079eba-a60c-4397-8182-c2a36d25c664)
